app/controllers/idv/in_person/ssn_controller.rb

```diff
@@ -0,0 +1,106 @@
+module Idv
+  module InPerson
+    class SsnController < ApplicationController
+      include IdvStepConcern
+      include StepIndicatorConcern
+      include StepUtilitiesConcern
+      include Steps::ThreatMetrixStepHelper
+      include ThreatMetrixConcern
+
+      before_action :renders_404_if_in_person_ssn_info_controller_enabled_flag_not_set
+      before_action :confirm_verify_info_step_needed
```

Comment:
I'm surprised to see this here -- my impression is that it redirects the user to the review page unless the user has completed the verify info step. And users won't have completed the verify info step when they're on the SSN page

And yet it seems like it's working! I see it on the idv SSN page. Can you explain this bit to me?

Reply:
@sheldon-b  Hmmm... I initially thought I added it to handle an edge case (on edit, maybe?!????). I cannot think of any reason it is needed. Commenting it out and testing proves it is not needed, just as your predicted. 

I deleted `before_action :confirm_verify_info_step_needed` with commit 405ca4e8b256a381e7138bd0ec9a1ca29fa359c5

Reply:
@gina-yamada I'm sorry I actually mixed up the logic. It redirects users who have completed the verify_info page to the review page. So if a user completes the verify_info page they can't navigate backwards to the ssn page. That makes sense. Will you add it back please? Woops!

Reply:
Added back in. Commit 8c227917d72eac4057838566b36d1cb48271831e